third_party/blink/renderer/modules/storage/storage_area.cc

```diff
@@ -164,6 +164,16 @@ bool StorageArea::Contains(const String& key,
 
 void StorageArea::NamedPropertyEnumerator(Vector<String>& names,
                                           ExceptionState& exception_state) {
+  if (!cached_area_->memory_used() &&  // This means either empty or not loaded
+      v8::recordreplay::IsReplaying() &&
+      v8::recordreplay::AreEventsDisallowed()) {
+    // TODO: Use `IsReplayCode` instead -
+    // https://linear.app/replay/issue/RUN-1502
+
+    // Ignore side effects during interceptor key collection to execute during
+    // pauses. See: https://linear.app/replay/issue/RUN-1315#comment-26f96699
```

Comment:
this comment doesn't make sense here, can you update it to a more appropriate one?